CombineUnloadAIDriver.lua

```diff
@@ -1260,7 +1260,7 @@ function CombineUnloadAIDriver:startDrivingToCombine()
 			-- (short) pathfinding to get under the pipe.
 			zOffset = - self:getCombinesMeasuredBackDistance() - 10
 		else
-			zOffset = - self:getCombinesMeasuredBackDistance() - 2
+			zOffset = - self:getCombinesMeasuredBackDistance() - 10 --allow trailer space to align after sharp turns (noticed it more affects potato/sugarbeet harvesters with pipes close to vehicle)
```

Comment:
would it make sense then to make this as a function of the combine's pipe offset? 
```lua
self:getPipeOffset(self.combineToUnload)
```

should work to get the offset.